modules/home/default.nix

```diff
@@ -119,9 +119,13 @@ in {
     initExtra = "${shellScriptDefault}/bin/${shellScriptDefault.name}";
     shellInit = {
       fish.interactiveShellInit = initExtra;
-    } // genAttrs ["zsh" "bash"] (_: {
-      inherit initExtra;
-    });
```

Comment:
previously this would assign `shellInit.bash` and `shellInit.zsh`, so this change needs to go inside that attrset or it does nothing but remove the configs entirely ><